src/libraries/System.Diagnostics.TraceSource/src/System.Diagnostics.TraceSource.csproj

```diff
@@ -29,11 +29,15 @@
     <Compile Include="System\Diagnostics\SwitchLevelAttribute.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="System.Collections.NonGeneric" />
```

Comment:
why are you removing the P2Ps?

Reply:
I reverted all but system.io.filesystem bcoz it doesnt have any ridless tfm

Reply:
I'm fine if you want to switch to named references (it makes sense for shared framework parts) but I just wanted to understand the reason for this change.